[CI] [low prio] Two workflow failures on fork after master PR #420

After syncing my fork to master, two workflows failed:

<img width="647" height="284" alt="Image" src="https://github.com/user-attachments/assets/8f518908-175f-4a21-b09b-f15791c1b869" />

Both jobs were re-run and resulted in the same errors on my side.


## For Docker Publish 
Failed at the "Build and push" substage of the workflow. Last lines read as follows:

```
#14 exporting to image
#14 pushing layers 0.6s done
#14 ERROR: failed to push ghcr.io/scaleovenstove/crosswithfriends:latest: denied: permission_denied: The requested installation does not exist.
------
 > exporting to image:
------
ERROR: failed to build: failed to solve: failed to push ghcr.io/scaleovenstove/crosswithfriends:latest: denied: permission_denied: The requested installation does not exist.
Reference
Check build summary support
Error: buildx failed with: ERROR: failed to build: failed to solve: failed to push ghcr.io/scaleovenstove/crosswithfriends:latest: denied: permission_denied: The requested installation does not exist.
```

I tested changing line 34 of `.github\workflows\docker-publish.yml` to `images: ghcr.io/${{ github.repository_owner }}/crosswithfriends` and that seemed to work on my fork. It may be a fix, but I'm not sure how it impacts any other workflows (if at all).


## For Post-Deploy Test: Seed Testing
Failed at the "Seed load test data" substage of the "Seed testing database" job. Error reads as follows:

```
Run psql "$DATABASE_URL" -f loadtest/seed.sql
psql: error: connection to server on socket "/var/run/postgresql/.s.PGSQL.5432" failed: No such file or directory
	Is the server running locally and accepting connections on that socket?
Error: Process completed with exit code 2.
```

Seems like `$DATABASE_URL` is empty; most likely missing a secret and needs a workaround - maybe skip seed on forks that don't have the secret?

--- a/.github/workflows/deploy-tests.yml
+++ b/.github/workflows/deploy-tests.yml
@@ -55,7 +55,12 @@ jobs:
       - name: Seed load test data
         env:
           DATABASE_URL: ${{ secrets.TESTING_DB_URL }}
-        run: psql "$DATABASE_URL" -f loadtest/seed.sql
+        run: |
+          if [ -z "$DATABASE_URL" ]; then
+            echo "Skipping — TESTING_DB_URL secret not configured (fork?)"
+            exit 0
+          fi
+          psql "$DATABASE_URL" -f loadtest/seed.sql
 
   playwright:
     name: Playwright E2E

--- a/.github/workflows/docker-publish.yml
+++ b/.github/workflows/docker-publish.yml
@@ -32,7 +32,7 @@ jobs:
         id: meta
         uses: docker/metadata-action@v5
         with:
-          images: ghcr.io/scaleovenstove/crosswithfriends
+          images: ghcr.io/${{ github.repository_owner }}/crosswithfriends
           tags: |
             type=raw,value=latest,enable={{is_default_branch}}
             type=semver,pattern={{version}}

